### ISSUE TITLE
Wrong Deposit and Withdraw Info.

### ISSUE THREAD

**Issue by bilalhassan555** [gateio/gateapi-python#120]
https://api.gateio.ws/api/v4/spot/currencies
In this endpoint, MDX_BSC withdrawal is enabled.
![gateio_api](https://user-images.githubusercontent.com/70362148/173871131-a04888d3-bbe2-4537-a5bb-4a190f463f85.png)

But in website,
![mdx_withdraw_page](https://user-images.githubusercontent.com/70362148/173871217-61767950-1897-420d-a3b6-d3be1e7ca21d.png)

No BSC chain present in this!!!!.
10SET, POLC, KTN - here are some examples in BSC chain that says deposit or withdrawal is available.

KABY, CNAME - here are some examples in MATIC chain that says deposit or withdrawal is available.

The list is not ending here. these are some examples. 
This is so serious that API is misguiding users by providing false information.

**Comment by revilwang** [comment]
Yes, we're working on this to ensure the API's configuration is consistent with what's shown on the web page.

**Comment by bilalhassan555** [comment]
> Yes, we're working on this to ensure the API's configuration is consistent with what's shown on the web page.

hey revilwang, 
Its more than two months now. No correction is made. You are spreading misinformation here regarding deposit and you are responsible to release the token if anyone deposit a token via API information.

**Comment by revilwang** [comment]
Sorry for the trouble. Actually the issue has been addressed several times .I'll contact the implementation to make sure what causes the problem continuously appearing.

**Comment by bilalhassan555** [comment]
> Sorry for the trouble. Actually the issue has been addressed several times .I'll contact the implementation to confirm what causes the problem continuously appearing.

May I know how many Years you need to FIX this simple issue???
